packages/convertim/src/Controller/LoginController.php

```diff
@@ -52,4 +65,42 @@ public function getLoginDetail(Request $request): Response
             return $this->convertimLogger->logGenericException($e);
         }
     }
+
+    /**
+     * @param \Symfony\Component\HttpFoundation\Request $request
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    #[Route('/login')]
+    public function login(Request $request): Response
+    {
+        if (!$this->isConvertimEnabled()) {
+            return $this->convertimNotEnabledResponse();
+        }
+
+        try {
+            /** @var \Lcobucci\JWT\Token\Plain $token */
```

Comment:
the inline annotation is not necessary, or? From the types, we know the method returns `\Lcobucci\JWT\Token` interface and that seems to be sufficient as we call just `$token->claims()` and that method is defined on the interface